tests/operator/operator.go

```diff
@@ -2326,23 +2326,41 @@ var _ = Describe("[sig-operator]Operator", Serial, decorators.SigOperator, func(
 				Expect(cert3.Leaf.NotAfter).To(BeTemporally("<", now))
 
 				By("Adding the first cert")
-				cm.Data[components.CABundleKey] = string(cert1Encoded)
-				_, err = virtClient.CoreV1().ConfigMaps(flags.KubeVirtInstallNamespace).Update(context.Background(), cm, metav1.UpdateOptions{})
-				Expect(err).ToNot(HaveOccurred())
+				Eventually(func() error {
+					cm, err = virtClient.CoreV1().ConfigMaps(flags.KubeVirtInstallNamespace).Get(context.Background(), components.ExternalKubeVirtCAConfigMapName, metav1.GetOptions{})
+					if err != nil {
+						return err
+					}
+					cm.Data[components.CABundleKey] = string(cert1Encoded)
+					_, err = virtClient.CoreV1().ConfigMaps(flags.KubeVirtInstallNamespace).Update(context.Background(), cm, metav1.UpdateOptions{})
+					if err != nil {
```

Comment:
nit - Just return `err` here and the same below?

Reply:
Done